packages/postman/utils/string-utils.ts

```diff
@@ -0,0 +1,9 @@
+export function camelToSnake(str) {
+  return str.replace(/[A-Z]/g, (match) => `_${match.toLowerCase()}`)
+}
+
+export const snakeToTitleCase = (str) => {
```

Comment:
Would be great to have both functions in the same declaration style:

```suggestion
export function snakeToTitleCase(str: string) {
```